src/utils/PdfGenerator/HBSCompiler.ts

```diff
@@ -0,0 +1,14 @@
+import * as hbs from "handlebars"
```

Comment:
czy tak nie zadziała? 
```suggestion
import hbs from "handlebars"
```

Reply:
może `handlebars` będzie jednak lepszą opcja niż skrótowiec, zwłaszcza że użyte jest tylko raz a łatwiej ogarnąć w kodzie że chodzi o `handlebars` hbs nie wiele mówi :)